src/iqm/benchmarks/__init__.py

```diff
@@ -26,7 +26,6 @@
     BenchmarkRunResult,
 )
 from .circuit_containers import BenchmarkCircuit, CircuitGroup, Circuits
-from .compressive_gst.compressive_gst import CompressiveGST, GSTConfiguration
```

Comment:
Could you pull the latest changes from `main` into your branch?

There was a PR a few days ago handling the import issue.